packages/components/src/components/tooltip/tooltip.custom.styles.ts

```diff
@@ -0,0 +1,8 @@
+import { css } from 'lit';
+
+export default css`
```

Comment:
We have new tokens exactly for the tooltip. Shouldn`t we override the current style configuration to use them?

Reply:
They are already used in the tooltip.styles.ts